engines/decidim-debates/decidim-debates.gemspec

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+$LOAD_PATH.push File.expand_path("../lib", __FILE__)
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |s|
+  s.name        = "decidim-debates"
+  s.summary     = "A debates component for decidim's participatory processes."
+  s.description = s.summary
+  s.version     = "0.0.1"
+  s.authors     = ["Josep Jaume Rey Peroy", "Marc Riera Casals", "Oriol Gual Oliva", "Genís Matutes Pujol"]
+  s.email       = ["josepjaume@gmail.com", "mrc2407@gmail.com", "oriolgual@gmail.com", "genis.matutes@gmail.com"]
```

Comment:
It's not wrong but we've omitted these so it "inherits" it from the main gem, which is the only one published.